javascript/javascript1/week2/weather.js

```diff
@@ -0,0 +1,14 @@
+function clothesBaseTemperature(temperature) {
+  if (temperature <= 20) {
+    return "Jackets and thermals";
+  } else if (temperature > 20 && temperature < 25) {
+    return "Denim jacket and Jeans";
+  } else if (temperature >= 25 && temperature < 40) {
+    return "Short sleeve Tshirts and Pants";
+  } else if (temperature >= 40) {
+    return "cotton clothes and linen";
+  }
```

Comment:
What if the temperature is more or less than 100 degrees?

Reply:
ok..I will add that condition also